charts/rancher-logging-example/charts/log-output/templates/deployment.yaml

```diff
@@ -31,6 +30,10 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 80
```

Comment:
Do we want this to be override-able? Overkill feature, maybe

Reply:
The more ridged the service the better.  It removes possible failure points. It is not designed to run publicly so users can customize the port on port forward.